octopoes/octopoes/models/ooi/findings.py

```diff
@@ -1,13 +1,30 @@
+import abc
+from enum import Enum
 from typing import Literal, Optional
 
+from pydantic import AnyUrl
+
 from octopoes.models import OOI, Reference
 from octopoes.models.persistence import ReferenceField
 
 
-# todo: make abstract
-class FindingType(OOI):
+class RiskLevelSeverity(Enum):
+    CRITICAL = "critical"
+    HIGH = "high"
+    MEDIUM = "medium"
+    LOW = "low"
+    RECOMMENDATION = "recommendation"
+
+
+class FindingType(OOI, abc.ABC):
     id: str
 
+    description: Optional[str]
+    source: Optional[AnyUrl]
+
+    risk_score: Optional[float]
+    risk_severity: Optional[RiskLevelSeverity]
```

Comment:
Do these make more sense on a FindingType versus on a Finding? I can imagine two Findings having a different risk_severity depending on which object they point to. (E.g. A `RetireJSFindingType` on your main website has higher risk than on third party websites found by KAT.) In that case risk attributes on the FindingTypes could also be interpreted as defaults.

But then again I have not gone into the risk modelling that much yet so far.

Reply:
Findings like the KAT findings currently have a static score that is used for all the findings that are created and we need to have a place to store those. They are currently stored in the rocky database, but we are moving those to xtdb and thus need the fields also there.

Reply:
Yes seems legitimate, but I was just questioning the fact that all Findings of the same type have the same risk score and severity as there is no way to differentiate the risk between two findings of the same type now.

Reply:
@Donnype 

I think your point is valid, a Finding of a certain FindingType might be more or less severe depending on its relation to the Organisation. However, this is a separate use-case and needs more thought on how, when and where to implement this functionality. 

For now, as @dekkers says, this story concerns 'just' moving the FindingType metadata to XTDB/Octopoes to improve performance of several views and the reports.